mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -712,6 +722,9 @@ class SparseVector @Since("1.0.0") (
     }
   }
 
+  @Since("1.6.0")
+  private[spark] def foreach(f: (Double) => Unit) = { indices.foreach { i: Int => f(values(i)) } }
```

Comment:
It adds the ability to easily do foreach including non-active values on the vector.